internal/pkg/operand/strategy.go

```diff
@@ -0,0 +1,24 @@
+package operand
+
+// DeletionStrategy describes how to handle operands on-cluster when deleting the associated operator.
+type DeletionStrategy string
```

Comment:
It would be nice to make this type implement the `pflag.Value` interface. Then we could plug it in directly and we could perform validation during flag parsing.

Reply:
That sounds great